.github/workflows/playwright.yml

```diff
@@ -15,6 +15,20 @@ jobs:
     - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6
       with:
         node-version: lts/*
+    - name: Setup Minio
+      run: |
+          docker run -d -p 9000:9000 --name minio \
```

Comment:
Instead of using a sidecar container defined at the action level, it turns out you can just run raw docker commands from inside the action, which is what I'm doing here.